### PR TITLE
pacman-key: add page

### LINKED ISSUE #6294: page request: pacman-key

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Closes #6294

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/pacman-key.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`sudo pacman-key --add {{path/to/keyfile}}`
+
+- Receive a key from a key server:
+
+`sudo pacman-key --recv-keys "{{key_id}}"`
+
+- Print the fingerprint of a specific key:
+
+`pacman-key --finger "{{key_id}}"`
```
Comment (sbrl) [inline pages/linux/pacman-key.md:28]: Not sure I understand the point of this point. If you're specifying the key id, isn't that the fingerprint already?

File: pages/linux/pacman-key.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Sign an imported key locally:
+
+`sudo pacman-key --lsign-key "{{key_id}}"`
```
Comment (sbrl) [inline pages/linux/pacman-key.md:32]: Sign with what?

File: pages/linux/pacman-key.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/pacman-key.md:28]: No, based on the `pacman-key --list-keys`, the `uid` is `first_name last_name <me@example.com>`.
So, You can use `pacman-key --finger me@example.com` or `pacman-key --finger 'first_name last_name'` to see the fingerprint.

File: pages/linux/pacman-key.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/pacman-key.md:32]: From the Arch Wiki <https://wiki.archlinux.org/title/Pacman/Package_signing>:
> you must locally sign the imported key:
> # pacman-key --lsign-key keyid
> You now trust this key to sign packages.

From <https://www.gnupg.org/gph/en/manual/r1020.html>:
> lsign-key — locally sign a public key with a private key

I think this is similar to trust an imported key. :thinking: Looks like a common practice.

File: pages/linux/pacman-key.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/pacman-key.md:28]: Ah I see. Thanks for the explanation! I wonder if we could rephrase the `key_id` at all?

File: pages/linux/pacman-key.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# pacman-key
+
+> Wrapper script for GnuPG used to manage pacman’s keyring.
```
Comment (marchersimon) [inline pages/linux/pacman-key.md:3]:
```suggestion
> Wrapper script for GnuPG used to manage pacman's keyring.
```

File: pages/linux/pacman-key.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`sudo pacman-key --populate {{archlinux}}`
+
+- List keys from the public keyring:
+
+`pacman-key --list-keys`
+
+- Add the specified keys:
+
+`sudo pacman-key --add {{path/to/keyfile}}`
```
Comment (marchersimon) [inline pages/linux/pacman-key.md:20]: Is this correct?
```suggestion
`sudo pacman-key --add {{path/to/keyfile.gpg}}`
```

**Review by marchersimon — APPROVED** [review]
Thanks!

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**